.travis.yml

```diff
@@ -113,6 +113,7 @@ script:
       if [ "$B" = "cmake" ]; then
         mkdir bin
         cd tests
+        docker pull ghcr.io/libssh2/ci_tests_openssh_server || true
```

Comment:
The next line needs to be adjusted to use the same image name, otherwise it will create a second image. You could also re-tag the pulled image locally to the existing internal image name with `docker tag`.

Reply:
It will create a new image, but that should not be a problem because the new image will use the pulled image's cached layers.